projects/marketing-site/components/SimpleNavbar/MobileNavbar.tsx

```diff
@@ -0,0 +1,106 @@
+import {  Switch, Menu, Transition } from '@headlessui/react'
+import { MoonIcon, ArrowRightOnRectangleIcon, GlobeAltIcon, ChevronRightIcon, CheckIcon,
+        UserGroupIcon, CubeIcon, PresentationChartBarIcon, BriefcaseIcon } from '@heroicons/react/20/solid'
+import { useTheme } from 'next-themes';
+import React, { useState, Fragment } from 'react';
+import commonLang from "@/lang/common.json";
+import { useRouter } from 'next/router';
+import NavigationMain from '@/libs/interface/navigation';
+import Link from 'next/link';
+
+
+const navigation = [
+    { name: "vision", href: "/", icon: BriefcaseIcon },
+    { name: "team", href: "/team", icon: UserGroupIcon },
+    { name: "funding", href: "/funding", icon: PresentationChartBarIcon },
+    { name: "token", href: "/token", icon: CubeIcon },
+  ];
+  
+
+export default function MobileNavbar() {
+  const { theme, setTheme } = useTheme();
+  const [enabled, setEnabled] = useState(theme === 'dark' ? true : false)
+
+  const { locale, locales, asPath } = useRouter();  
+
+  return (
+    <div className="space-y-2 py-6 text-sm">
+        {navigation.map((item) => (
+            <div key={item.name}>
+                <Link href={item.href} className="-mx-3 rounded-lg py-1 px-3 font-semibold
+                    leading-7 dark:text-gray-100 text-black inline-flex">
+                    <item.icon className="h-6 w-6 mr-2"/>
+                    {commonLang.menu.filter(j => j.locale === locale)[0][item.name as keyof NavigationMain]} 
```

Comment:
the more that i look at the lines of code that put the text on the site, the more that i think we should create a helper function to make this more readable. something like `{translate(locale, {'menu', 'name'})}` or something like that where the function parameters are the path to the text.